src/Polytopes/Polyhedron/constructors.jl

```diff
@@ -60,16 +56,21 @@ pm_polytope(P::Polyhedron) = P.pm_polytope
 
 ### Construct polyhedron from V-data, as the convex hull of points, rays and lineality.
 @doc Markdown.doc"""
-    convex_hull(V [, R [, L]])
+    convex_hull(V::Matrix [, R::Matrix [, L::Matrix]]; non_redundant::Bool = false)
 
 The polytope given as the convex hull of the rows of a set of points.
 
-see Def. 2.11 and Def. 3.1  of [JT13](@cite)
+The matrices rows are the points, representative vectors of the rays and the generators of the lineality space,
```

Comment:
"matrices rows" makes no sense. I guess you could write "matrix rows", but the sentence is then still confusing overall. It would be much clearer if the arguments were referenced by names and the sentence split in three.... or perhaps `V` could already be referenced in the first sentence of this docstring. Then the bit about `representative vectors of the rays`, which I assume is about `R`, could be merged with the bit on `R` that comes next, and likewise for `L`.

Also, you don't really say what happens if `L` is missing?